packages/tenant-process/test/integration/getTenantVerifiedAttributeVerifiers.test.ts

```diff
@@ -0,0 +1,287 @@
+import {
+  Tenant,
+  generateId,
+  TenantId,
+  AttributeId,
+  VerifiedTenantAttribute,
+  tenantAttributeType,
+  TenantVerifier,
+  attributeKind,
+} from "pagopa-interop-models";
+import {
+  getMockContext,
+  getMockTenant,
+  getMockAttribute,
+} from "pagopa-interop-commons-test";
+import {
+  addOneTenant,
+  addOneAttribute,
+  tenantService,
+} from "../integrationUtils.js";
+import {
+  getMockVerifiedTenantAttribute,
+  getMockCertifiedTenantAttribute,
+} from "../mockUtils.js";
+
+describe("getTenantVerifiedAttributeVerifiers", () => {
+  const tenantId: TenantId = generateId();
+  const attributeId: AttributeId = generateId();
+  const verifier1Id: TenantId = generateId();
+  const verifier2Id: TenantId = generateId();
+
+  const verifier1: TenantVerifier = {
+    id: verifier1Id,
+    verificationDate: new Date("2024-01-01T10:00:00Z"),
+    expirationDate: undefined,
+    extensionDate: undefined,
+  };
+
+  const verifier2: TenantVerifier = {
+    id: verifier2Id,
+    verificationDate: new Date("2024-01-02T10:00:00Z"),
+    expirationDate: undefined,
+    extensionDate: undefined,
+  };
+
+  const verifiedAttribute: VerifiedTenantAttribute = {
+    ...getMockVerifiedTenantAttribute(),
+    id: attributeId,
+    assignmentTimestamp: new Date("2024-01-01"),
+    verifiedBy: [],
+    revokedBy: [],
+  };
+
+  const tenant: Tenant = {
+    ...getMockTenant(),
+    id: tenantId,
+    attributes: [verifiedAttribute],
+  };
+
+  beforeEach(async () => {
+    await addOneTenant({
+      ...getMockTenant(),
+      id: verifier1Id,
+    });
+    await addOneTenant({
+      ...getMockTenant(),
+      id: verifier2Id,
+    });
+    await addOneAttribute({
+      ...getMockAttribute(attributeKind.verified),
+      id: attributeId,
+    });
+  });
+  it("should retrieve verifiers for a verified attribute", async () => {
+    const tenantWithVerifiers: Tenant = {
+      ...getMockTenant(),
+      id: tenantId,
+      attributes: [
+        {
+          ...getMockVerifiedTenantAttribute(),
+          id: attributeId,
+          assignmentTimestamp: new Date("2024-01-01"),
+          verifiedBy: [verifier1, verifier2],
+          revokedBy: [],
+        },
+      ],
+    };
+
+    await addOneTenant(tenantWithVerifiers);
+
+    const result = await tenantService.getTenantVerifiedAttributeVerifiers(
+      tenantId,
+      attributeId,
+      { offset: 0, limit: 10 },
+      getMockContext({})
+    );
+
+    expect(result.results).toHaveLength(2);
+    expect(result.totalCount).toBe(2);
+
+    expect(result.results[0]).toEqual({
+      id: verifier1Id,
+      verificationDate: verifier1.verificationDate,
+      expirationDate: undefined,
+      extensionDate: undefined,
+      delegationId: undefined,
+    });
+
+    expect(result.results[1]).toEqual({
+      id: verifier2Id,
+      verificationDate: verifier2.verificationDate,
+      expirationDate: undefined,
+      extensionDate: undefined,
+      delegationId: undefined,
+    });
```

Comment:
Same as other comment

```suggestion
    expect(result).toEqual({
      totalCount: 2,
      results: [ ... ]
     })
```